tests/e2e/test_signing.js

```diff
@@ -54,105 +52,33 @@ describe("Wormhole tests", () => {
   });
 
   it("test shim signing for zome call", async () => {
-    const payload = await getPayload(keys);
-    loggedActivity = await appWs.callZome({
-      cell_id: [Buffer.from(slCellId[0]), Buffer.from(slCellId[1])],
-      zome_name: 'service',
-      fn_name: 'log_activity',
-      payload,
-      cap: null,
-      provenance: Buffer.from(slCellId[1])
-    });
+    console.log("Calling zome test...", testCellId);
+    try {
+      loggedActivity = await appWs.callZome({
+        cell_id: [Buffer.from(testCellId[0]), Buffer.from(testCellId[1])],
+        zome_name: 'test',
+        fn_name: 'returns_obj',
+        payload: null,
+        cap: null,
+        provenance: Buffer.from(testCellId[1])
+      });
+      console.log("return from signing test: ", loggedActivity);
+      expect(loggedActivity).to.be.ok
+    } catch(e) {
+      console.log("Failing...", e);
+      expect(false).to.be.ok
+    }
   });
 
 });
 
-const sign = (keys, data) => {
-  let msg_bytes = msgpack.encode(data)
-  const sig_bytes = keys.sign(msg_bytes);
-  return sig_bytes
-}
-
-async function getPayload(keys) {
-  let request_payload = {
-    call_spec: {
-      args_hash: "uhCkkmrkoAHPVf_eufG7eC5fm6QKrW5pPMoktvG5LOC0SnJ4vV1Uv",
-      function: "get_message",
-      zome:"chat",
-      dna_alias: "element-chat",
-      hha_hash: "uhCkkmrkoAHPVf_eufG7eC5fm6QKrW5pPMoktvG5LOC0SnJ4vV1Uv"
-    },
-    host_id: "d5xbtnrazkxx8wjxqum7c77qj919pl2agrqd3j2mmxm62vd3k",
-    timestamp: [162303,0]
-  }
-
-  let request = {
-    agent_id: Codec.AgentId.encode(keys.publicKey()),
-    request: request_payload,
-    request_signature: await sign(keys, request_payload)
-  }
-
-  let response_payload = "kRW1wN0luUmxjM1FpT2lKcGJtWnZjbTFoZEdsdmJpSXNJblJvYVhNaU9uc2liblZ0WW1WeUlqb3hMQ0ozYVd4c0lqcGJJbUpsSUd"
-  let response = {
-    response_hash: response_payload,
-    host_metrics: {
-      cpu: 7,
-      bandwidth: 1
-    },
-    signed_response_hash: await sign(keys, response_payload),
-    weblog_compat: {
-      source_ip: "100:0:0:0",
-      status_code: 200
-    }
-  }
-
-  let confirmation_payload = {
-    response_digest: "JblJvYVhNaU9uc2liblZ0WW1WeUlqb",
-    metrics: {
-      response_received: [165303,0]
-    }
-  }
-  let confirmation = {
-    confirmation: confirmation_payload,
-    confirmation_signature: await sign(keys, confirmation_payload)
-  }
-  return {
-    request,
-    response,
-    confirmation
-  }
-}
-
-async function setUpServicelogger(appWs) {
```

Comment:
Why are we deleting so much of this file? Are we still testing signing and servicelogger thoroughly without this code?

Reply:
This test was just to test making a zomeCall with the shim, So the current test is much cleaner.